extensions/amp-story/1.0/bookend/components/heading.js

```diff
@@ -37,8 +37,8 @@ export class HeadingComponent {
    * @override
    * */
   assertValidity(headingJson) {
-    user().assert('text' in headingJson, 'Heading component must contain' +
-      ' `text` field, skipping invalid.');
+    user().assert('text' in headingJson, 'Heading component must contain ' +
+      '`text` field, skipping invalid.');
```

Comment:
Nit: `` `'text' field, skipping invalid.` `` instead of ``'`text` field, skipping invalid.'``

Reply:
As chatted offline our ES linter doesn't allow us to do so 😬